vignettes/examples.Rmd

```diff
@@ -16,4 +16,146 @@ knitr::opts_chunk$set(
+  print_to_gt(demog_data)
+```
+
+While this make a table it isn't a very nice table and defiantly doesn't match the mock. So lets start with formatting all the numbers. To do this we are going to build a `body_plan` to add to our `tfrmt`.  This will be a fairly quick explanation of `body_plan`s but if you like more information see `vignettes("Body Plan")` 
+
+Body plans are made up of a series of `frmt_stucture`s where each `frmt_stucture` represents the formatting of a cell within the table. This will always be applies latest to oldest when applying formatting. This means the first `frmt_stucture` in the `body_plan` should be the most generic. You can use the groups, labels and parameters to specify which formatting applies to which values.  
+
+To start with we are going to use all the rows that are "n (%)" as the default. This way we don't need to list out every row that is an "n (%)" row. These rows are made up of two different values, so we will need to use `frmt_combine`. Next, we can format the continuous variables, which is just a straight forward one value per row so we can just use the label to filter and `frmt` to define the look. Finally, we want format the p-values. This is a bit more complicated because although the group and label will match the other values it needs it own formatting. To so this we will need to specify the parameter in the `frmt_structure` like so `frmt_structure(group_val = ".default", label_val = ".default", p = frmt("x.xx")`. But, we also here we need to make sure it never reports 0 or 1. So we can use `frmt_when` to specify the formatting based on the value. 
+```{r}
+tfrmt(
+  # specify columns in the data
+  group = vars(rowlbl1,grp),
+  label = rowlbl2,
+  column = column, 
+  param = param,
+  values = value,
+  sorting_cols = vars(ord1, ord2),
+  # specify value formatting 
+  body_plan = body_plan(
+    frmt_structure(group_val = ".default", label_val = ".default", frmt_combine("{n} ({pct} %)", 
+                                                                                n = frmt("xxx"),
+                                                                                pct = frmt("xx.x"))),
+    frmt_structure(group_val = ".default", label_val = "n", frmt("xxx")),
+    frmt_structure(group_val = ".default", label_val = c("Mean", "Median", "Min","Max"), frmt("xxx.x")),
+    frmt_structure(group_val = ".default", label_val = "SD", frmt("xxx.xx")),
+    frmt_structure(group_val = ".default", label_val = ".default", p = frmt_when(">0.99" ~ ">0.99",
+                                                                                 "<0.001" ~ "<0.001",
+                                                                                 TRUE ~ frmt("x.xxx", missing = "")))
+  )) %>% 
+  print_to_gt(demog_data)
+```
+Now the numbers looks all correct, we can drop the order columns and the `grp` column. To do this we use a `col_plan` which uses `tidy-select` nomenclature to drop/move columns. 
```

Comment:
```suggestion
Now that the numbers are formatted correctly, we can drop the order columns and the `grp` column. To do this we use a `col_plan` which uses `tidy-select` nomenclature to drop/move columns. 
```

Reply:
I suggest talking about the col plan last after we have used the `grp` variable for the spacing